ConsumerApi/ConsumerApi.csproj

```diff
@@ -21,6 +21,8 @@
 		<PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.9" />
 		<PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.18.1" />
 		<PackageReference Include="ReHackt.Extensions.Options.Validation" Version="7.0.1" />
+		<PackageReference Include="Serilog.AspNetCore.Enrichers.CorrelationId" Version="1.0.0" />
```

Comment:
As I've written in another comment, the enricher developed by Serlilog doesn't seem to be maintained. So we should probably switch back to the 3rd party one.

Reply:
Unbelievable...

How about https://github.com/serilog-contrib/serilog-enrichers-clientinfo? This library has support for a CorrelationId as well:

```c#
.WithCorrelationId(headerName: "correlation-id", addValueIfHeaderAbsence: true)
```

Reply:
Looks good, thanks!